### ISSUE TITLE
Test VoiceOver support. 

### ISSUE THREAD

**Issue by cambardell** [cambardell/Footnote2#4]

**Comment by Taenerys** [comment]
Hey Cameron! I'd love to help with this issue 😊 Super cool app by the way! 👍

**Comment by cambardell** [comment]
Yes please, thanks Taenerys! It would be awesome if you could either create issues with potential VoiceOver fixes, or just fix them yourself if you're comfortable with that and open a PR. Contact me any time 👍
